### PR TITLE
كان عندي خطاء في الارسال

### PR DESCRIPTION
**Pull request by yasminbasim**

### CODE REVIEW

File: src/solution_part2.kt [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+ val lifePontPlayer1=100
+ val player2="player2"
+ val lifePointPlyer2=45
+
+ if (lifePontPlayer1>=90) {
+  println("player $player1 your health is fine")
+ } else {
+  println("player $player1  your healthing ")
+ }
+ if (lifePointPlyer2>= 50) {
```
Comment (shuruqt1000) [inline src/solution_part2.kt:13]: logic error

**Comment by shuruqt1000** [comment]
good job

File: src/solution_part2.kt (hunk 1 above)
Comment (yasminbasim) [reply src/solution_part2.kt:13]: if (lifePontPlayer1>90) {
  println("player $player1 your health is fine")
 } else {
  println("player $player1  your healthing ")
 }

**Comment by shuruqt1000** [comment]
completed

**Pull request closed without merging**